packages/react-table/build-single-packages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "packageName": "@patternfly/react-table",
```

Comment:
Can this package name not be inferred from where this config is being executed?

Reply:
I would prefer to leave it this way in case in the future someone else needs this script like in extensions.  They would need to pass in the name.